var/spack/repos/builtin/packages/py-astropy/package.py

```diff
@@ -19,6 +19,7 @@ class PyAstropy(PythonPackage):
 
     license("BSD-3-Clause")
 
+    version("6.1", sha256="6c3b915f10b1576190730ddce45f6245f9927dda3de6e3f692db45779708950f")
```

Comment:
Spack cannot do the proper version substitution without this fix:

```suggestion
    version("6.1.0", sha256="6c3b915f10b1576190730ddce45f6245f9927dda3de6e3f692db45779708950f")
```

Since the `6.1` release is technically flagged as `6.1.0` (see https://pypi.org/project/astropy/6.1.0/).

Reply:
Made the version 6.1.0